scripts/tei.p5.xsl

```diff
@@ -36,6 +36,14 @@
 <!-- ==================================================================== -->
 <!--                            OVERRIDES                                 -->
 <!-- ==================================================================== -->
+  
+  <!-- Adding links to other cases via bibls https://github.com/CDRH/earlywashingtondc/issues/124 -->
+  
+  <xsl:template match="/TEI/text//bibl[@type='case'][@xml:id]">
+    <a href="{@xml:id}" class="tei_bibl_link">
```

Comment:
I'm trying to think of any instance of one case linking to another where the URL structure would change. they are all this link format, correct? http://earlywashingtondc.org/cases/oscys.caseid.0251